package.json

```diff
@@ -4,11 +4,14 @@
   "description": "node_service_adlib-backend",
   "main": "app.js",
   "scripts": {
-    "start": "node app.js"
+    "dev": "nodemon --exec babel-node src/app.js",
+    "build": "babel src --out-dir dist --copy-files",
+    "start": "node dist/app.js",
+    "lint": "eslint src/"
   },
   "dependencies": {
     "cookie-parser": "^1.4.5",
-    "correlation-id": "^4.0.0",
+    "correlation-id": "^3.1.1",
```

Comment:
waarom downgraden?

Reply:
De downgrade staat ook niet in de .lock file:
```
    "correlation-id": {
      "version": "4.0.0",
      "resolved": "https://registry.npmjs.org/correlation-id/-/correlation-id-4.0.0.tgz",
      "integrity": "sha512-WvXtJBlovvOBKqTz/YwWP2gm6CXJZJArfGimp9s/ehmhJMPFbmnPMQe3K60Q9idGNixMvKojMjleyDhZEFdHfg==",
      "requires": {
        "uuid": "^8.3.1"
      }
    },

```

Reply:
Downgrade aangezien we momenteel node 10 images gebruiken en de app breekt. Enkel v3 is compatibel. https://github.com/toboid/correlation-id#compatibility 
De .lock file moet idd geupdate worden. Ik heb waarschijnlijk geen 'npm install' gerund aangezien ik dit teste via docker build